src/examples/src/widgets/speed-dial/Basic.tsx

```diff
@@ -11,13 +11,6 @@ export default factory(function Basic({ middleware: { icache } }) {
 	return (
 		<Example>
 			<SpeedDial>
-				<Action
-					onClick={() => {
-						icache.set('action', 'Mailing');
-					}}
-				>
-					<Icon type="mailIcon" />
-				</Action>
```

Comment:
Since we're using a sandboxed example with a default "medium" size, one action was removed from each example so things fit without scrolling and without explicit heights.